src/xrServerEntities/script_ini_file.h

```diff
@@ -8,7 +8,12 @@
 
 #pragma once
 
+#include "pch_script.h"
```

Comment:
NEVER do this. This file should only be included into .cpp's which need this file.

Reply:
Okay, removed. But for what this header is needed?

Reply:
pch_script.h is a precompiled header